rama-cluster/single/start.sh

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env bash
+
+
+sudo yum update -y
+
+echo "Starting zookeeper..." >> setup.log
+
+sudo cp zookeeper.service /etc/systemd/system &>> setup.log
+sudo systemctl start zookeeper.service &>> setup.log
+sudo systemctl enable zookeeper.service &>> setup.log
+
+# If Zookeeper successfully starts, we don't need this log anymore
+rm setup.log
```

Comment:
What is setup.log for? Also, how does removing it here do anything if the background processes are still outputting to it?

Reply:
Yea, let's remove all of that.